config/labels/analyzers/clang-tidy.json

```diff
@@ -3456,7 +3485,10 @@
     ],
     "clang-diagnostic-mismatched-new-delete": [
       "doc_url:https://clang.llvm.org/docs/DiagnosticsReference.html#wmismatched-new-delete",
-      "severity:MEDIUM"
+      "guideline:sei-cert",
+      "profile:security",
+      "sei-cert:mem51-cpp",
+      "severity:HIGH"
```

Comment:
Ditto.

Reply:
Can cause runtime issues, that's with it is a high severity issue.